lib/jekyll-import/importers/medium.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module JekyllImport
+  module Importers
+    class Medium < Importer
+      def self.specify_options(c)
+        c.option "username",       "--username NAME",  "Medium username"
+        c.option "canonical_link", "--canonical_link", "Copy original link as canonical_url to post (default: false)"
+        c.option "render_audio",   "--render_audio",   "Render <audio> element in posts for the enclosure URLs (default: false)"
+      end
+
+      def self.validate(options)
+        abort "Missing mandatory option --username." if options["username"].nil?
+      end
+
+      def self.require_deps
+        Importers::RSS.require_deps
+      end
+
+      def self.process(options)
+        Importers::RSS.process({
+          "source"         => "https://medium.com/feed/@#{options.fetch("username")}",
+          "render_audio"   => options.fetch("render_audio", false),
+          "canonical_link" => options.fetch("canonical_link", false),
+          # Add existing tags from Medium post to front matter using `category` subfield on the RSS <item>
+          "extract_tags"   => "category",
```

Comment:
Out of curiosity, why not support all of the RSS options? Should we just suggest to users on the docs page for Medium to use the RSS importer and show them how to use it?

Reply:
> Out of curiosity, why not support all of the RSS options?

If you see all the options from RSS importer is there except **tag**.

> Should we just suggest to users on the docs page for Medium to use the RSS importer and show them how to use it?

It's possible but it'll be little verbose. Currently this Medium importer is just the wrapper for RSS importer.